méli-mélo/th-empathy/style.css

```diff
@@ -0,0 +1,98 @@
+/* Thematic purple */
+.panel-empathy,
+hr.brdr-empathy {
+    border-color: #5C2790;
+}
+
+.btn-empathy.btn-primary {
+    background-color: #5C2790 !important;
+}
+
+/* Purple button hover and focus state */
+.btn-empathy.btn-primary:hover,
+.btn-empathy.btn-primary:focus,
+.btn-empathy.btn-primary:active {
+    background-color: #D6B9EC !important;
+    color: #FFF;
+}
+
+/* Light Purple button default state */
+.btn-empathy.btn-default {
+    background-color: #FFF !important;
+    border-color: #5C2790 !important;
+    color: #5C2790 !important;
```

Comment:
Optional but recommended, please remove all the `!important` as I don't they are needed considering the high specificity of your CSS selector.